_data-prepper/pipelines/configuration/processors/otel-trace-group.md

```diff
@@ -0,0 +1,84 @@
+
+### Amazon OpenSearch Service
+
+See the following example `YAML` configuration file for the `OTel trace group` processor when you use the Amazon OpenSearch Service:
+
+``` YAML
+pipeline:
+  ...
+  processor:
+    - otel_trace_group:
+        hosts: ["https://your-amazon-opensearch-service-endpoint"]
+        aws_sigv4: true
+        cert: path/to/cert
+        insecure: false
+```
+
+## Configuration
+
+You can configure the Amazon OpenSearch Service with the following options.
+
+<!--- Are we talking about the OTel processor or OpenSearch? This is confusing. --->
+
+| Name | Required | Description | Default value
+| -----| ----| -----------| -----|
+| `hosts`| Yes | A list of IP addresses of OpenSearch nodes. | Default value | 
+| `cert` | No | A certificate authority (CA) certificate that is PEM-encoded. Accepts both .pem or .crt. This enables the client to trust the CA that has signed the certificate that OpenSearch is using. | `null` |
+| `aws_sigv4` | No | A boolean flag to sign the HTTP request with AWS credentials. Only applies to Amazon OpenSearch Service. See [OpenSearch security](https://github.com/opensearch-project/data-prepper/blob/129524227779ee35a327c27c3098d550d7256df1/data-prepper-plugins/opensearch/security.md) for details. | `false`. |
+| `aws_region` | No | A String that represents the region of Amazon OpenSearch Service domain, for example, `us-west-2`. Only applies to Amazon OpenSearch Service. | `us-east-1` |
+| `aws_sts_role_arn`| No | An identity and access management (IAM) role that the sink plugin assumes to sign the request to the Amazon OpenSearch Service. If not provided, the plugin uses the (Default credentials)[https://sdk.amazonaws.com/java/api/latest/software/amazon/awssdk/auth/credentials/DefaultCredentialsProvider.html]. | `null` |
+| `aws_sts_header_overrides` | No | A map of header overrides to make when assuming the IAM role for the sink plugin. | `null` |
+| `insecure` | No | A boolean flag to turn off SSL certificate verification. If set to `true`, CA certificate verification is turned off and insecure HTTP requests are sent. | `false` |
+| `username` | No | A string that contains the username and is used in the [internal users](https://opensearch.org/docs/latest/security/access-control/users-roles/) of your OpenSearch cluster. | `null` |
+| `password` | No | A string that contains the password and is used in the [internal users](https://opensearch.org/docs/latest/security/access-control/users-roles/) of your OpenSearch cluster. | `null` |
+
+## Configuration option examples
+
+You can define the `aws_sts_header_overrides` configuration option values. See the following example.
```

Comment:
@dlvenable @asifsmohammed This looks like it's missing information for context. Are these parameters in a YAML file?

Reply:
`aws_sts_header_overrides `is an option in the OTel trace group processor. You can see it in line 56.